src/lib/components/layout/Sidebar/SearchInput.svelte

```
@@ -1,226 +1,29 @@
 <script lang="ts">
-	import { getAllTags } from '$lib/apis/chats';
-	import { tags } from '$lib/stores';
-	import { getContext, createEventDispatcher, onMount, onDestroy, tick } from 'svelte';
-	import { fade } from 'svelte/transition';
+	import { getContext, createEventDispatcher } from 'svelte';
+	import SearchIcon from '$lib/components/icons/Search.svelte';
+	import SearchModal from './SearchModal.svelte';
 
-	const dispatch = createEventDispatcher();
 	const i18n = getContext('i18n');
+	const dispatch = createEventDispatcher();
```

ESLint

[error] 'dispatch' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)